### ISSUE TITLE
Improving README.md

### ISSUE THREAD

**Issue by niloysikdar** [niloysikdar/awesome-portfolio-template#10]
Need to add necessary content with some good styling in the README.md file.

**Comment by Devbhuti-Nandik** [comment]
Hello, I want to solve this issue as my first open-source contribution. Please provide the details and a bit of guidance regarding your issue. 
Thanks.

**Comment by niloysikdar** [comment]
Awesome @Devbhuti-Nandik . Glad to see you here. If you are a beginner then you can go through [this blog](https://niloysikdar.medium.com/how-to-make-your-first-pull-request-on-github-c76301e83557). Will be glad to help you with any further queries.

**Comment by Devbhuti-Nandik** [comment]
I have gone through the blog. So the next step is to fork your project and do the work only on your README.md file?

**Comment by niloysikdar** [comment]
Yes @Devbhuti-Nandik , first fork the project, then create your separate branch and then start working on the issue. Also, take some references from [the video](https://www.youtube.com/watch?v=eVGEea7adDM) if you want. Cheers.

**Comment by niloysikdar** [comment]
Hello @Devbhuti-Nandik , if you are facing any issue you can discuss it here.

**Comment by harshita2216** [comment]
I can help you on this issue.
